erpnext/vehicles/doctype/vehicle_booking_order/vehicle_booking_order.py

```diff
@@ -1040,3 +1054,68 @@ def update_allocation_booked(vehicle_allocation, is_booked, is_cancelled):
 	is_cancelled = cint(is_cancelled)
 	frappe.db.set_value("Vehicle Allocation", vehicle_allocation, {"is_booked": is_booked, "is_cancelled": is_cancelled},
 		notify=True)
+
+
+@frappe.whitelist()
+def send_customer_vehicle_anniversary_notifications():
+	if not automated_vehicle_anniversary_enabled():
+		return
+
+	now_dt = now_datetime()
+	date_today = getdate(now_dt)
+
+	reminder_dt = get_anniversary_scheduled_time(date_today)
+	if now_dt < reminder_dt:
+		return
+
+	notification_last_sent_date = frappe.db.get_default("vehicle_anniversary_notification_last_sent_date")
+	if notification_last_sent_date and getdate(notification_last_sent_date) == date_today:
+		return
+
+	vehicle_anniversary_data = get_vehicle_anniversary_data(date_today)
+
+	for d in vehicle_anniversary_data:
+		doc = frappe.get_doc("Vehicle Booking Order", d.name)
+		doc.send_customer_vehicle_anniversary_notification()
+
+	if vehicle_anniversary_data:
+		frappe.db.set_default("vehicle_anniversary_notification_last_sent_date", date_today)
+
+def automated_vehicle_anniversary_enabled():
+	from frappe.core.doctype.sms_settings.sms_settings import is_automated_sms_enabled
+	from frappe.core.doctype.sms_template.sms_template import has_automated_sms_template
+
+	if is_automated_sms_enabled() and has_automated_sms_template("Vehicle Booking Order", "Vehicle Anniversary"):
+		return True
+	else:
+		return False
+
+def get_anniversary_scheduled_time(date_today=None):
+	vehicle_settings = frappe.get_cached_doc("Vehicles Settings", None)
+
+	reminder_date = getdate(date_today)
+	reminder_time = vehicle_settings.vehicle_anniversary_time or get_time("00:00:00")
+	reminder_dt = combine_datetime(reminder_date, reminder_time)
+
+	return reminder_dt
+
+def get_vehicle_anniversary_data(date_today = None):
```

Comment:
rename method to `get_bookings_for_vehicle_anniversary_notification`
and
no spaces between argument and operator and value